CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ separate changelogs for each crate were used. If you need to refer to these old
 
 ## [Unreleased]
 
+### Fixed
+
+### Added
+
+- libcnb: Store struct now supports `clone()` and `default()`. ([#547](https://github.com/heroku/libcnb.rs/pull/547))
```

Comment:
Technically, this is `libcnb-data`. I'm unsure what's best to write here since most users (I assume) will not use `libcnb-data` directly and might not even be aware that it exists. So far, we don't have `libcnb-data` prefixed changelog entries and prefixes seem to have started just a couple of versions ago.

I'm fine keeping this as-is, this is as minor as it gets and there is not a clear best way to handle this. Just bringing this up in case someone from @heroku/languages has a better idea. :)

Reply:
I think perhaps we should use `libcnb-data: ` for these, since it's more precise. For example, if we were to make a change to `libcnb-package`, I don't think we would want to describe it as two entries, one for `libcnb-test` and one `libcnb-cargo`, but a single `libcnb-package` entry. I think the changelog being unified should prevent end users from being confused about whether they are affected or not, which was the reason we had the cross-linking thing before.

Reply:
Updated the changelog. I can go either way. 

FWIW I basically didn't even notice that's what was happening with `libcnb:` I just copied the next line down and edited it. With a question of tags or no tags. I'm not sure what we want to get out of them. I prefer to keep my changelog format lightweight (I even prefer to not make my links to PRs proper markdown links so it's one less thing to change/edit and one less thing to mess up and and have a weird git blame). Though you all know me and my tendency to make mistakes on small stuff like that.

I would say the value of the tag is in letting people filter for only libraries they're using. In that case I would say that `libherokubuildpack` seems to be the only odd one out, though there could be more. I would expect anyone else to be using all of libcnb. 

Merge is blocked on resolution of this comment thread